app/views/user_mailer/verify_email.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Congratulations <%= @user.name %> on successfully signing in on Flash Sales</h1>
+<h3>Please click on the below link to verify your email</h3>
+
+<%= link_to 'Verify your Email', "http://127.0.0.1:3000/users/verify/#{@user.verification_token}"%>
```

Comment:
there is an issue in this url. what is it?

Reply:
We can change the routes - for example, we changed /profile to /user then we will have to change the url string in the form as well but if were using a helper then it will be updated by utself

Reply:
Fixed issue